server/lib/transforms/html/video.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const match = require('@quarterto/cheerio-match-multiple');
+const url = require('url');
+
+const youtube = videoId => `<amp-youtube
+	data-videoid="${videoId}"
+	layout="responsive"
+	width="480" height="270">
+</amp-youtube>`;
+
+module.exports = match({
+	'a[href^="http://video.ft.com/"]'(el, i, $, {brightcoveAccountId, brightcovePlayerId} = {}) {
```

Comment:
This will be a bit invalid for new videos, since they're now ft.com/content/uuid 😢